test/isEmpty.js

```diff
@@ -1,47 +1,28 @@
 var assert = require('assert');
 
-var R = require('..');
+var isEmpty = require('..').isEmpty;
```

Comment:
Just removing duplicates. I often hear people speak about how ugly it is to prefix all functions with `R.`. Since the `R` namespace was only used to access `isEmpty` I thought I could do that only once.